contentcuration/contentcuration/tests/test_exportchannel.py

```diff
@@ -1,16 +1,24 @@
+import md5
```

Comment:
This file has the real fix to make our tests pass. The rest is just to set up a proper environment.

Reply:
it's the little things in life